image_transport/src/camera_publisher.cpp

```diff
@@ -172,6 +189,26 @@ void CameraPublisher::publish(
   impl_->info_pub_->publish(info);
 }
 
+void CameraPublisher::publish(
+  sensor_msgs::msg::Image::UniquePtr image,
+  sensor_msgs::msg::CameraInfo::UniquePtr info,
+  rclcpp::Time stamp) const
+{
+  if (!impl_ || !impl_->isValid()) {
+    // TODO(ros2) Switch to RCUTILS_ASSERT when ros2/rcutils#112 is merged
```

Comment:
This PR https://github.com/ros2/rcutils/pull/112 was closed, do you mind to remove this comment ?